packages/paste-website/src/components/tokens-list/index.tsx

```diff
@@ -47,6 +49,32 @@ interface TokensListProps {
 }
 
 const getTokensByTheme = (theme: ThemeVariants, props: TokensListProps): TokenCategory[] => {
+  const font = props.consoleTokens[0].node.tokens.find(ele => ele.categoryName === 'fonts');
+  if (font) {
+    font.info = (
+      <Callout variant="secondary">
+        <CalloutTitle as="h4">Heads up about fonts in Paste!</CalloutTitle>
+        <CalloutText>
+          Due to font-family licensing, we have no ability to redistribute certain themed fonts. Paste makes the
+          assumption that the Whitney fonts are loaded if you are using the Console theme, and Colfax for Sendgrid.
```

Comment:
`Paste leaves it up to you to load the fonts needed for the theme you've selected.`